packages/openid4vc/src/openid4vc-verifier/router/authorizationEndpoint.ts

```diff
@@ -26,10 +26,24 @@ export function configureAuthorizationEndpoint(router: Router, config: OpenId4Vc
       const authorizationResponse: AuthorizationResponsePayload = request.body
       if (isVpRequest) authorizationResponse.presentation_submission = JSON.parse(request.body.presentation_submission)
 
-      // FIXME: we should emit an event here and in other places
+      const verificationSession = await openId4VcVerifierService.findVerificationSessionForAuthorizationResponse(
+        agentContext,
+        {
+          authorizationResponse,
+          verifierId: verifier.verifierId,
```

Comment:
Related to previous comments. Prefixing the id prop with the entity name seems inconsistent with other parts of the codebase

Reply:
So as mentioned in previous comments. There is id and verifierId in the verifier record .... :/